cafebabel/static/js/articles-tags.js

```diff
@@ -171,8 +170,10 @@ class TagEventListener {
   static keyup() {
     const inputNewTag = document.querySelector('.tags input[name=tag-new]')
     inputNewTag.addEventListener('keyup', event => {
+      event.preventDefault()
       /* Return if arrow up, arrow down are pressed */
-      if (event.keyCode == 40 || event.keyCode == 38) return
+      if (event.keyCode == 13 || event.keyCode == 40 || event.keyCode == 38)
```

Comment:
Arrow up, down… and enter?
It would be nice to tell _why_ this hack needs to be processed in very short.

Reply:
Comment should be updated accordingly